templates/github/.github/workflows/release.yml.j2

```diff
@@ -285,19 +285,3 @@ jobs:
 
       - name: Stage changelog for {{ plugin_default_branch }} branch
         run: python .github/workflows/scripts/stage-changelog-for-default-branch.py {{ "${{ github.event.inputs.release }}" }}
```

Comment:
I think this step, the one above it (Cleanup repository before making changelog PR) and the workflow script (stage-changelog-for-default-branch.py) can all be removed as well.